docs/docs/setup/debs.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 2
+---
```

Comment:
`deploy.md` already has this `sidebar_position`. I don't know how exactly Docusaurus breaks ties in this situation, but when I built the docs this file was sorted before `deploy.md` in the sidebar which seems counter-intuitive to me. :thinking: 

This probably should be `3`, and the files with the subsequent `sidebar_position`-s moved up by one. It's ... suboptimal, but we don't know of a way to better handle this unfortunately. >_<

Reply:
Fixed.  Also corrected the rest of the `sidebar_position` values across the whole `setup` section while I was at it.  Didn't check to see if the rest had issues.